mobile/src/modules/auth/store/actions.ts

```diff
@@ -139,11 +130,7 @@ export const getAlias = createActionFn<string, Promise<Alias | undefined>>(
     const { nodeHost, apiRootUrl } = state.app.burstService.settings;
     // TODO: unify network request actions, add proper error handling and so on
     const api = composeApi(new ApiSettings(nodeHost, apiRootUrl));
-    try {
-      const alias = await api.alias.getAliasByName(account);
-      return alias;
-    // tslint:disable-next-line: no-empty
-    } catch (e) {}
+    return await api.alias.getAliasByName(account);
```

Comment:
couldn't this result in an uncaught exception, killing the app?

Reply:
the only call to this action is already wrapped into a `try-catch` block. I tested it thoroughly with valid and invalid aliases and had no blank screen on the app.

tbh, the use of a thunk for this call is not really correct, but this is another story.